manage.py

```
@@ -1,3 +1,4 @@
+
 #!/usr/bin/env python3
```

Ruff (0.15.12)

[warning] Avoid whitespace before shebang

Remove whitespace before shebang

(EXE004)